diff --git a/src/MainWindow.cpp b/src/MainWindow.cpp
index ab746d4e..09f81e83 100644
--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -675,7 +675,7 @@ bool MainWindow::queryClose()
 
 void MainWindow::saveProperties(KConfigGroup &group)
 {
-    _viewManager->saveSessions(group);
+    /* _viewManager->saveSessions(group); */
 }
 
 void MainWindow::readProperties(const KConfigGroup &group)
@@ -685,7 +685,7 @@ void MainWindow::readProperties(const KConfigGroup &group)
 
 void MainWindow::saveGlobalProperties(KConfig *config)
 {
-    SessionManager::instance()->saveSessions(config);
+    /* SessionManager::instance()->saveSessions(config); */
 }
 
 void MainWindow::readGlobalProperties(KConfig *config)
